### PR TITLE
source-concord contribution from aazam-gh

### LINKED ISSUE #45976: [connector-request] Concord
This is a **suggestion** to create a connector for **Concord** using the UI Connector Builder.
The website for the connector is https://www.concordnow.com.

If you want to try, claim this issue and start working on it following the steps below.

### Steps:
1. Comment in the issue and wait to be assigned to start working on it.
2. Map the API endpoints (get approval to move to 3 step)
3. Describe steps to get credentials.
4. Create the connector using the UI Builder

> [!IMPORTANT]
> Airbyte doesn't have sandbox credentials for this connector. You must have access/credentials to the service provider to create the connector.

> [!CAUTION]
> This is a **suggestion** and there may be cases where creating the connector using the Builder won't be possible. To minimize the risk of investing a lot of time directly in creating the connector, we strongly recommend to follow steps above.

### PR DESCRIPTION
**Pull request by aazam-gh**
## What

This PR adds a new connector for source Concord (source-concord).

Contributor-provided connector description:

This is the setup for the Concord source which ingests data from the concord API.

Concord turns contract data into financial insights. Sign, store and search unlimited contracts https://www.concord.app/

In order to use this source, you must first create a concord account and log in. Then navigate to Automations -&gt; Integrations -&gt; Concord API -&gt; Generate New Key to obtain your API key.

The API is accessible from two environments, sandbox and production. You can learn more about the API here https://api.doc.concordnow.com/
## Details

<details><summary>Expand for list of spec properties and streams</summary>

### Configuration

| Input | Type | Description | Default Value |
|-------|------|-------------|---------------|
| `api_key` | `string` | API Key.  |  |
| `enviornment` | `string` | enviornment. The environment from where you want to access the API. |  |

### Streams
| Stream Name | Primary Key | Pagination | Supports Full Sync | Supports Incremental |
|-------------|-------------|------------|---------------------|----------------------|
| agreements | uid | DefaultPaginator | ✅ |  ❌  |
| user_organizations | id | No pagination | ✅ |  ❌  |
| organization | id | No pagination | ✅ |  ❌  |
| folders | id | No pagination | ✅ |  ❌  |
| reports | id | DefaultPaginator | ✅ |  ❌  |
| tags | id | No pagination | ✅ |  ❌  |

</details>

## Reviewer checklist
- [ ] Run `/format-fix` to resolve any formatting errors
- [ ] Help the contributor upload an icon that follows our icon design guidelines
- [ ] Click into the CI workflows that wait for a maintainer to run them, which should trigger CI runs

### CODE REVIEW

File: docs/integrations/sources/concord.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+In order to use this source, you must first create a concord account and log in. Then navigate to Automations -&gt; Integrations -&gt; Concord API -&gt; Generate New Key to obtain your API key.
+
+The API is accessible from two environments, sandbox and production. You can learn more about the API here https://api.doc.concordnow.com/
+
+## Configuration
+
+| Input | Type | Description | Default Value |
+|-------|------|-------------|---------------|
+| `api_key` | `string` | API Key.  |  |
+| `enviornment` | `string` | enviornment. The environment from where you want to access the API. |  |
```
Comment (marcosmarxm) [inline docs/integrations/sources/concord.md:15]:
```suggestion
| `environment` | `string` | environment. The environment from where you want to access the API. |  |
```

Fix typo. Can you link to Concord docs where to find the info?

File: docs/integrations/sources/concord.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
+# Concord
+This is the setup for the Concord source which ingests data from the concord API.
+
+Concord turns contract data into financial insights. Sign, store and search unlimited contracts https://www.concord.app/
+
+In order to use this source, you must first create a concord account and log in. Then navigate to Automations -&gt; Integrations -&gt; Concord API -&gt; Generate New Key to obtain your API key.
```
Comment (marcosmarxm) [inline docs/integrations/sources/concord.md:6]:
```suggestion
In order to use this source, you must first create a concord account and log in. Then navigate to Automations -> Integrations -> Concord API -> Generate New Key to obtain your API key.
```

**Review by marcosmarxm — CHANGES_REQUESTED** [review]
Can you also implement organizations membership and groups?

**Comment by marcosmarxm** [comment]
Hello 👋 Due to the numerous contributions happening, I decided to create this [view](https://github.com/orgs/airbytehq/projects/104/views/3) to help you know when I plan to review your contribution. If your connector has some limited-time credential feel free to reach me out.

**Comment by aazam-gh** [comment]
@marcosmarxm Ready for review again

**Comment by aazam-gh** [comment]
> Can you also implement organizations membership and groups?

membership is added, groups would require team account, out of scope at the moment

**Review by marcosmarxm — APPROVED** [review]

**Pull request merged**